mainapp/templates/mainapp/camp_details.html

```diff
@@ -35,6 +35,14 @@ <h3>{{ message }}</h3>
         <br>
         ക്യാമ്പിന്റെ വിവരങ്ങൾ ഇവിടെ അപ്ഡേറ്റ് ചെയ്യുക  
     </p>
-  </form> 
+  </form>
+
+  <script type="text/javascript">
+  window.onload = function() {
```

Comment:
use `window.addEventListener('load', function`, just to make sure that no other code will overwrite the onload function.